src/bundle/Resources/views/parts/tag.html.twig

```diff
@@ -0,0 +1,20 @@
+ <div class="ez-tag" 
+    {% for attribute_name, attribute_value in tag_attributes %}
```

Comment:
Not sure it this is good idea/solution to handle attributes - enables adding custom attributes. What do you think? ping @mikadamczyk @ViniTou @adamwojs

Reply:
@mikadamczyk @ViniTou @adamwojs can you respond here?

Reply:
It might be a good idea if there is a need for that but if you decided to make an extension point then refactor `class="ez-tag"` to use `tag_attributes` array :) 

To make sure class is always added you can do:
```twig
{% set tag_attributes = tag_attributes|default({})|merge({'class': (tag_attributes.class|default('') ~ ' ez-tag')|trim}) %}
```